api/managementpb/backup/artifacts.proto

```diff
@@ -16,6 +16,8 @@ enum BackupStatus {
   BACKUP_STATUS_PAUSED = 3;
   BACKUP_STATUS_SUCCESS = 4;
   BACKUP_STATUS_ERROR = 5;
+  BACKUP_STATUS_DELETING = 6;
+  BACKUP_STATUS_FAILED_TO_DELETE = 7;
```

Comment:
How we should handle this status? How artifact can be moved to another state from this status?

Reply:
I'm setting `BACKUP_STATUS_DELETING` at the beginning and then I'm setting `BACKUP_STATUS_FAILED_TO_DELETE ` if for some reason removal from S3 has failed, or removing db entries on success. In case of `BACKUP_STATUS_FAILED_TO_DELETE` user can retry removing.

Reply:
If you set `BACKUP_STATUS_DELETING` and the server was restarted, on startup server will try to finish removal? Or user able to retry removal?